services/rook-ceph-cluster/1.10.1/defaults/cm.yaml

```diff
@@ -0,0 +1,139 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rook-ceph-cluster-1.10.1-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |
+    ---
+    operatorNamespace: ${releaseNamespace}
+    clusterName: dkp-ceph-cluster
+    toolbox:
+      # If needed, enable a toolbox for debugging (creates a pod with ceph CLI)
+      enabled: false
+
+    # PSP is Unsupported in 1.25+ k8s
+    # Set this to the same value as the rook-ceph chart.
+    pspEnable: false
+
+    # All values below are taken from the CephCluster CRD
+    cephClusterSpec:
+      cephVersion:
+        image: quay.io/ceph/ceph:v17.2.3
+      dataDirHostPath: /var/lib/rook
+
+      mon:
+        count: 3
+        # Mons should only be allowed on the same node for test environments where data loss is acceptable.
+        allowMultiplePerNode: false
+        volumeClaimTemplate:
+          spec:
+            # Use the default storage class configured in cluster.
+            storageClassName: ""
+            volumeMode: FileSystem
+            resources:
+              requests:
+                storage: 10Gi
+
+      mgr:
+        count: 2
+        allowMultiplePerNode: false
+
+      dashboard:
+        enabled: true
+        urlPrefix: ""
+        port: 8443
+        ssl: false
+
+      storage:
+        storageClassDeviceSets:
+          - name: rook-ceph-osd-set1
+            count: 3
+            portable: false
+            encrypted: false
+            placement:
+              topologySpreadConstraints:
+              - maxSkew: 1
+                topologyKey: topology.kubernetes.io/zone # The nodes in the same rack have the same topology.kubernetes.io/zone label.
+                whenUnsatisfiable: ScheduleAnyway
+                labelSelector:
+                  matchExpressions:
+                    - key: app
+                      operator: In
+                      values:
+                        - rook-ceph-osd
+                        - rook-ceph-osd-prepare
+              - maxSkew: 1
+                topologyKey: kubernetes.io/hostname
+                whenUnsatisfiable: ScheduleAnyway
+                labelSelector:
+                  matchExpressions:
+                    - key: app
+                      operator: In
+                      values:
+                        - rook-ceph-osd
+                        - rook-ceph-osd-prepare
+            volumeClaimTemplates:
+              - metadata:
+                  name: data
+                spec:
+                  resources:
+                    requests:
+                      storage: 20Gi
```

Comment:
so this means 3 PVs at 20 GI each? but that is split between all the services that use this storage aka Loki and Velero? 

we might need to bump the default but I think the plan was to revisit this cluster config anyway in another ticket

(sorry if we discussed this already 🙈)

Reply:
yep, we ll test and increase this once we actually integrate loki, velero and once we have some concrete data we can adjust this. nothing is set in stone with this PR, we just dont have enough data to come up with an ideal value.